cw-orch-mock/src/core.rs

```diff
@@ -298,46 +307,32 @@ impl<S: StateInterface> TxHandler for Mock<S> {
             .map_err(From::from)
     }
 
-    fn wait_blocks(&self, amount: u64) -> Result<(), CwOrchError> {
+    fn wait_blocks(&self, amount: u64) -> Result<(), CwEnvError> {
         self.app.borrow_mut().update_block(|b| {
             b.height += amount;
             b.time = b.time.plus_seconds(5 * amount);
         });
         Ok(())
     }
 
-    fn wait_seconds(&self, secs: u64) -> Result<(), CwOrchError> {
+    fn wait_seconds(&self, secs: u64) -> Result<(), CwEnvError> {
         self.app.borrow_mut().update_block(|b| {
             b.time = b.time.plus_seconds(secs);
             b.height += secs / 5;
         });
         Ok(())
     }
 
-    fn next_block(&self) -> Result<(), CwOrchError> {
+    fn next_block(&self) -> Result<(), CwEnvError> {
         self.app.borrow_mut().update_block(next_block);
         Ok(())
     }
 
-    fn block_info(&self) -> Result<cosmwasm_std::BlockInfo, CwOrchError> {
+    fn block_info(&self) -> Result<cosmwasm_std::BlockInfo, CwEnvError> {
         Ok(self.app.borrow().block_info())
     }
 }
 
-impl<T: CwOrchExecute<Mock> + ContractInstance<Mock> + Clone> CallAs<Mock> for T {
-    type Sender = Addr;
-
-    fn set_sender(&mut self, sender: &Addr) {
-        self.as_instance_mut().chain.sender = sender.clone();
-    }
-
-    fn call_as(&self, sender: &Self::Sender) -> Self {
-        let mut contract = self.clone();
-        contract.set_sender(sender);
-        contract
-    }
-}
-
```

Comment:
IS there a blanketed impl on this? Why was it removed?

Reply:
https://github.com/AbstractSDK/cw-orchestrator/pull/152#discussion_r1268616244
Yep!